enterprise/migrations/0076_auto_20190918_2037.py

```diff
@@ -0,0 +1,35 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.11.24 on 2019-09-18 20:37
+from __future__ import unicode_literals
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('enterprise', '0075_auto_20190916_1030'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='enterprisecustomer',
+            name='enable_portal_reporting_config_screen',
+            field=models.BooleanField(default=False, help_text='Specifies whether to allow access to the reporting configurations screen in the admin portal.'),
+        ),
+        migrations.AddField(
+            model_name='historicalenterprisecustomer',
+            name='enable_portal_reporting_config_screen',
+            field=models.BooleanField(default=False, help_text='Specifies whether to allow access to the reporting configurations screen in the admin portal.'),
+        ),
+        migrations.AlterField(
+            model_name='enterprisecustomer',
+            name='learner_portal_hostname',
```

Comment:
Why are we altering this field?

Reply:
not sure (this was auto generated) could this have happened because I didn't put the new field at the end of the model?

Reply:
Looks like Doug might have ran the migration, then changed the help text in this commit (https://github.com/edx/edx-enterprise/commit/0166b59b121bf1ae5e958c583ba5f3b93a2fbf84), so this is fine to ignore.